index.js

```diff
@@ -0,0 +1,152 @@
+const setup = require("./setup.js");
+const seedData = require("./seed-data.js");
+const sqlite = require("sqlite3").verbose();
+const db = new sqlite.Database("./poll.db");
+let command = process.argv.slice(2);
+
+//command = ["insert","politicians","rubhi","R","SF","9.123"]
+
+if(command.length != 0){
+    switch(command[0]){
+        case "setup":{
+            setup();
+            break;
+        }
+        case "seed":{
+            seedData();
+            break;
+        }
+        case "insert":{
+            create(command.slice(1))
+            break;
+        }
+        case "update":{
+            updateData(command[1],command[2],command[3],command[4]);
+            break;
+        }
+        case "delete":{
+            deleteData(command[1],command[2]);
+            break;
+        }
+        default:
+        console.log("Invalid command")
+            let help = "node index.js setup ==> for creating table\n";
+            help += "node index.js seed ==> for inserting dummy data to table\n";
+            help += "node index.js insert <politicians|voters|votes> <par1:name|firstName|voterID> <par2:party|lastName|politicianID> <par3:location|gender> <par4:grade|age> ==> for creating table\n";
+            help += "node index.js update <tableName> <column> <value> <id>\n";
+            help += "node index.js delete <tableName> <id>\n";
+            console.log(help);      
+    }
+}else{
+    // releasee 5
+    db.serialize(function(){
+        let query1=
+        `SELECT name,party,grade_current FROM politicians
+        WHERE party = 'R' AND grade_current BETWEEN 9 AND 11;`;
+    
+        let query2 = 
+        `
+        SELECT count(*) as 'totalVote', politicians.name from votes join politicians on 
+        votes.politician_id = politicians.id 
+        where politician_id = (
```

Comment:
kenapa tidak langsung saja
```suggestion
        where politicians.name = 'Olympia Snowe'
```